action.yml

```diff
@@ -9,5 +9,7 @@ runs:
         channel: stable
         flutter-version: 3.7.8
     - name: Install Shorebird
-      run: curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | sh
+      run: |
+        curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | sh
+        echo "/home/runner/.config/shorebird/bin" >> $GITHUB_PATH
```

Comment:
I don't think it's safe to hardcode this path because it will change depending on the type of runner (macos vs ubuntu vs windows)

The install directory is determined via:

```bash
install_dir() {
  [ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.shorebird" || printf %s "${XDG_CONFIG_HOME}/shorebird"
}
```

Reply:
Good call, fixed